gufe/components/smallmoleculecomponent.py

```diff
@@ -5,8 +5,29 @@
 logger = logging.getLogger('openff.toolkit')
 logger.setLevel(logging.ERROR)
 from openff.toolkit.topology import Molecule as OFFMolecule
+
+
+def _old_OFF() -> bool:
+    from openff.toolkit import __version__ as OFF_VERSION
+    a, b, c = map(int, OFF_VERSION.split('.'))
+    return (a, b) < (0, 11)
```

Comment:
This will error on import if a user is using a release candidate. If we'l need this with any regularity, the thing to use is `packaging.version.Version`.